about.html

```diff
@@ -38,33 +38,75 @@
+                <p class="about_text two">A casual scientific meeting to discuss the future of the biotechnology field with ponents from more than 27 countries is taking place in October, In New York.</p>
+                <p class="about_site">Please contact us via Email for any further questions about World Biotechnology Conference 2023!</p>
+                <h2 class="about_mail"> <u>worldbiotechsummit2023@email.com</u> > </h2>
             </div>
         </section>
+
+        <section class="logo_section">
+            <h2 class="logo_title">World Biotech Conference 2023 Logo</h2>
+            <hr class="about_line">
+            <p class="logo_text two">The world biotechnology conference 2023 logo was choosen in 2022.</p>
+            <p class="logo_text one">The world biotechnology conference 2023 logo orem ipsum dolor sit amet consectetur adipisicing elit. Autem, corrupti. Rem repudiandae a magni corrupti numquam modi, itaque necessitatibus delectus ipsum recusandae quo illo, eum velit iure saepe, nemo mollitia.</p>
+            <div class="logo_container">
+                <img class="logo" src="./assets/icons/biotech_logo_xl.png" alt="Logo">
+            </div>
+        </section>
+
+        <section class="logo_section">
+            <h2 class="logo_title">See the past World Biotech Conferences</h2>
+            <hr class="about_line">
+            <p class="logo_text">Take a look at the last two World Biotech Conferences which took place in Buenos Airse and Warsaw.</p>
+            <div class="past_img_container">
+                <div class="past_img_one">
+                    <h2 class="past_title">2019</h2>
+                    <p class="past_text">World Biotech Conference in Buenos Aires</p>
+                </div>
+                <div class="past_img_two">
+                    <h2 class="past_title">2021</h2>
+                    <p class="past_text">World Biotech Conference in warsaw</p>
+                </div>
+            </div>
+        </section>
+
     </main>
 
     <footer id="partners">
-        <div class="last_footer">
-            <figure class="footer_logo"></figure>
+        <div class="about_partners_container partners_container">
+            <h2 class="progam_section_title">Partner</h2>
+            <hr class="program_line">
+            <div class="partners">
+                <img class="partner_one pr" src="./assets/partners/partner_one.png"></img>
+                <img class="partner_three pr" src="./assets/partners/partner_three.png"></img>
+                <img class="partner_six pr" src="./assets/partners/partner_six.png"></img>
+                <img class="partner_two pr" src="./assets/partners/partner_two.png"></img>
+                <img class="partner_four pr" src="./assets/partners/partner_four.png"></img>
```

Comment:
- [ ] Kindly for screenreaders and accessibility needs add an `alt` tag to each `img` of your project. In this way the user will know what's going on if he/she is using a screenreader or the img isn't loaded in some browser 🥇